firebase-installations/src/main/java/com/google/firebase/installations/local/PersistedInstallationEntry.java

```diff
@@ -41,6 +41,9 @@
 
   public abstract long getTokenCreationEpochInSecs();
 
+  @Nullable
+  public abstract String getFISException();
```

Comment:
are we exposing this to clients? Having exception == String is weird. How about just calling it error message?

Reply:
Yes, the stored error message. Changed it. PTAL.

Idea: getId is non-blocking call. After we return the id, we encounter some error during registration. We persist the error message & return it to the clients next time getId is called.